### ISSUE TITLE
Cleanup untagged Docker images

### ISSUE THREAD

**Issue by benoit74** [openzim/_python-bootstrap#43]
Docker images keeps pilling up in the GHCR, since we have no solution to clean them up.

We should probably implement https://github.com/vlaurin/action-ghcr-prune to automatically clean the untagged images at the end of each run of the PublishDevDockerImage workflow.

**Comment by rgaudin** [comment]
Strongly in favor. We should also apply it to non-bootstrap projects

**Comment by benoit74** [comment]
Of course, I've just opened the issue here first to ensure we are aligned before deploying the approach in the wild ^^

**Comment by benoit74** [comment]
Looks like it was working fine, but this was indeed not matching documentation so I've replaced the comma with a space. I've also removed the cut-off date.

Unfortunately we will need to list all image names, grant access, and hence not forget to update to the same as new images are created ...

@rgaudin is it a go for you to deploy on all openzim, kiwix and offspot Docker images?

**Comment by rgaudin** [comment]
Good for me
